backend/src/test/kotlin/org/loculus/backend/controller/submission/SubmitProcessedDataEndpointTest.kt

```diff
@@ -615,6 +628,71 @@ class SubmitProcessedDataEndpointTest(
             )
     }
 
+    @Test
+    fun `WHEN I submit file for already approved entry THEN new file is public`() {
```

Comment:
The title of the test is a bit odd to me - as a user I cannot submit files for already approved entries, I have to make a revision. IMO it'd be good to mention the pipeline and pipeline version aspect. Maybe: 

```suggestion
    fun `WHEN the preprocessing pipeline submits files for already approved entries THEN new file is public`() {
```

(line might no be too long and cause lint failures)